src/pages/index.js

```diff
@@ -84,26 +104,153 @@ editCardPopup.setEventListeners();
 const userInfo = new UserInfo({
   titleSelector: ".profile__title",
   descriptionSelector: ".profile__description",
+  avatar: ".profile__image",
 });
 
+/*Delete card confirmation*/
+
+const deleteConfirmPopup = new PopupWithConfirmation({
+  popupSelector: "#modal-delete-card",
+  handleFormSubmit: () => {},
+});
+deleteConfirmPopup.setEventListeners();
+
+/*Edit Avatar*/
+
+const avatarProfilePopup = new PopupWithForm(
+  "#profile-avatar-modal",
+  avatarSubmit
+);
+avatarProfilePopup.setEventListeners();
+
+const EditAvatarBtn = document.querySelector(".profile__avatar-button");
```

Comment:
* Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)